src/app/mail/mail-header/mail-header.component.html

```diff
@@ -8,15 +8,23 @@
     <div class="row align-items-center mailbox-navbar-row">
         <div class="col-sm-6 mailbox-navbar-col hidden-xs-down">
             <div class="form search-form">
-                <form>
+                <div>
```

Comment:
Why not use `form` and then handle single `submit` event instead of handling `keyup.enter` and `click` separately?

Reply:
its because we have two different buttons inside the input, and submit was triggering the clear button.